src/funcCoreTools/getNpmDistTag.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// tslint:disable-next-line:no-require-imports
+import request = require('request-promise');
```

Comment:
Just FYI, in some scenarios I had issues with lint using request-promise because it uses bluebird.  In docker I switched to request-native, which is the same without bluebird.  You may not hit this, though, I think it was related to my trying to strongly type something from request.

Reply:
No I hit it, too. I worked around it by casting to `<Thenable<string>>` below.

I thought about moving to the request-native one you mentioned (after I saw your PR in docker), but I'm actually hoping we can switch to [request-light](https://www.npmjs.com/package/request-light), which is catered to VS Code extensions and supports proxies. I filed an issue here to get more info on the package: https://github.com/Microsoft/node-request-light/issues/4

Reply:
Cool, thanks